src/components/marker-table/ContextMenu.js

```diff
@@ -159,8 +159,46 @@ class MarkersContextMenu extends PureComponent<Props> {
     if (selectedMarker === null) {
       return;
     }
-
-    copy(markers[selectedMarker].name);
+    const marker = markers[selectedMarker];
+    let name = marker.name;
+
+    if (marker.data) {
+      const data = marker.data;
+      let category = 'unknown';
+      if (typeof data.category === 'string') {
+        category = data.category;
+      }
+      switch (data.type) {
+        case 'tracing':
+          if (category === 'log') {
```

Comment:
We don't need to create a `category` variable. We can use `data.category` directly here.